### PR TITLE
Orion IPC Screen

### PR DESCRIPTION
**Pull request by Stev-the-third**
yeah thats it self explanatory.
IPC Orion Face

### CODE REVIEW

**Comment by Stev-the-third** [comment]
MAN I FORGOT THE CHANGELOG DAMNIT

**Comment by Stev-the-third** [comment]
!review

**Comment by Stev-the-third** [comment]
!wip

**Comment by Stev-the-third** [comment]
im adding an ee face too

**Comment by Stev-the-third** [comment]
!review

File: html/changelogs/Stev-the-third-orion-face.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, this gets changed to [] after reading.  Just remove the brackets when you add new shit.
+# Please surround your changes in  double quotes ("). It works without them, but if you use certain characters it screws up compiling. The quotes will not show up in the changelog.
+changes:
+  imageadd: "Adds an Orion Express screen for Baseliners."
```
Comment (hazelrat) [inline html/changelogs/Stev-the-third-orion-face.yml:58]: Seems to be missing the dash, should resolve tests failing once fixed.

Comment (Stev-the-third) [reply html/changelogs/Stev-the-third-orion-face.yml:58]: aough. im dumb as hell

**Review by hazelrat — APPROVED** [review]

**Review by DreamySkrell — APPROVED** [review]

**Comment by Stev-the-third** [comment]
yeesh. my vsc did weird shit but i think i managed to wrestle it back. don't question it

**Pull request closed without merging**